website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,9 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2024-3-26
+* Remove beta tags from [run task stages](/terraform/cloud-docs/api-docs/run-tasks/run-task-stages-and-results) and [run tasks](/terraform/cloud-docs/api-docs/run-tasks/run-tasks).
+
```

Comment:
Do we generally include changes like this in the changelog? I did a look and couldn't find anything similar, but wanted to check in case I missed anything.

Reply:
TBH I don't know but it _felt_ wrong to remove the beta tags without a notification in the changelog.

Reply:
Although.... I should probably make it more obvious where it got removed e.g. ".... for the Global Run Tasks feature..."

Reply:
ping @BrianMMcClain

Reply:
Hey @glennsarti! So sorry, totally missed the notification for your messages a few days ago. My two cents is lets remove this from the changelog. Flipping through a few recent features that have exited beta, I'm not seeing any similar entries in the changelog.

Reply:
I'll remove it